test/acceptance/open_and_close.robot

```diff
@@ -63,3 +63,29 @@ Closing all browsers clears cache
     Run Keyword And Expect Error
     ...    No browser with index or alias 'Browser 2' found.
     ...    Switch Browser    Browser 2
+
+Click Element Without Modifier
+    Open Browser    ${ROOT}/jquery.html    ${BROWSER}
+    Set Selenium Implicit Wait    60s
+    Click Element    link=Link with id
+    Sleep    5s
+    Location Should Be    ${ROOT}/broken.html
+
+Click Element Open In New Window
+    Open Browser    ${ROOT}/jquery.html    ${BROWSER}
+    Set Selenium Implicit Wait    60s
+    Click Element    link=Link with id    SHIFT
+    Sleep    5s
+    @{windows}=    List Windows
+    Log To Console    ${windows}       
+    Select Window    ${windows[1]}
+    Location Should Be    ${ROOT}/broken.html
+
+Click Element With Incorrect Keys
+    Open Browser    ${ROOT}/jquery.html    ${BROWSER}
+    Set Selenium Implicit Wait    60s
+    ${error_message}=    Set Variable    KeyError: "'SHIFT+' click is not allowed with this keyword"
+    Run Keyword And Expect Error    ${error_message}    Click Element    link=Link with id    SHIFT+
+    ${error_message}=    Set Variable    KeyError: "'SHIFT+ALT' click is not allowed with this keyword"
+    Run Keyword And Expect Error    ${error_message}    Click Element    link=Link with id    SHIFT+ALT
```

Comment:
I do not see reason why we should block combinations which may or may not work. I think this should actually work.

Reply:
SHIFT+ALT + Click - > actually tries to save the link with Save window
SHIFT+ -> Is the incorrect way of specifying keys

So such things dont make sense as our keyword is more focused about opening link either in same window or new tab or new window